iphone/Classes/TiUIWindowProxy.m

```diff
@@ -304,6 +304,7 @@ - (void)viewWillAppear:(BOOL)animated;    // Called when the view is about to ma
 - (void)viewDidAppear:(BOOL)animated;     // Called when the view has been fully transitioned onto the screen. Default does nothing
 {
 	[self updateTitleView];
+    [self updateHidesBars];
```

Comment:
![image](https://cloud.githubusercontent.com/assets/839700/17167753/0852e0ae-53df-11e6-9083-6d97d2ca87ca.png)

I think someone broke the indentation with tabs.